_source/user-guide/alerts/configure-alerts.md

```diff
@@ -10,101 +10,166 @@ contributors:
   - imnotashrimp
 ---
 
-To create a new alert, you'll start in Kibana so you can test the query you want to use. After that, you'll continue to the Create a New Alert page, where you can configure the alert and notification settings.
+When you set out to create an alert,
+you'll need to start with a query.
+If you haven't formed your query yet,
+we recommend starting off in Kibana
+so you can see the results of the query in real time.
+
+Otherwise, if you already have a query and filters,
+you can create a blank alert
+from the top menu (**Alerts & Events > New alert**)
+or from the _Alert definitions_ page
```

Comment:
It's also available from the `triggered alerts` page

Reply:
Creating an alert? Not that I see. Only for editing.